ioc.types.ts

```diff
@@ -0,0 +1,24 @@
+const TYPES = {
+    AuthManager: Symbol('AuthService'),
+    Bucket: Symbol('Bucket'),
+    Container: Symbol('Container'),
+    DataReader: Symbol('IngestionReader'),
+    DataWriter: Symbol('IngestionWriter'),
+    DummyHandler: Symbol('DummyHandler'),
+    DummyMessage: Symbol('DummyMessage'),
+    INGEST: Symbol('Ingest'),
```

Comment:
It's a little odd seeing actions here mixed in with classes that are injectable via inversify. Perhaps you could give me a brief primer in how INGEST and PREVIEW are being used here.

Reply:
In the index.ts we're also using the container to resolve to a corresponding class that does have dependencies on things like sqs. It felt like a natural fit

Reply:
In the index.ts we're also using the container to resolve to a corresponding class that does have dependencies on things like sqs. It felt like a natural fit